wasm/tools/fixtures/runner.py

```diff
@@ -340,18 +332,6 @@ def get_command_fn(command):
 # properly refactored but for now they are skipped to allow for incremental
 # improvement to the library.
 SKIP_COMMANDS = {
-    'custom.wast': {
-        # It appears that the logic for parsing a binary module is broken,
-        # allowing the parser to exit before fully parsing the module.  For
-        # this test it ends up exiting before encountering the invalid section
-        # which **should** error out due to an invalid section ID.
-        93: Failed,
-        # This one appears to be lack of validation of the lengths of code and
-        # function section types.
-        # See the note here about malformed modules:
-        # - https://webassembly.github.io/spec/core/bikeshed/index.html#code-section%E2%91%A0
-        102: Failed,
-    },
```

Comment:
:tada: :tada:

Reply:
Yeah, this felt like really nice validation that the new structure is *correct*.  When I finished implementing the stream parser and unwrapped a few subtle transcription errors, these just started failing (i.e. passing when they were expected failures).